specification/dartLangSpec.tex

```diff
@@ -2126,22 +2126,25 @@ \section{Classes}
 are its static and instance members.
 
 \LMHash{}%
-A class has several scopes:
+A class declaration has two scopes:
 \begin{itemize}
-\item A \IndexCustom{type-parameter scope}{scope!type parameter},
+\item
+  A \IndexCustom{type-parameter scope}{scope!type parameter},
   which is empty if the class is not generic (\ref{generics}).
-The enclosing scope of the type-parameter scope of a class is the enclosing scope of the class declaration.
-\item A \IndexCustom{static scope}{scope!static}.
-The enclosing scope of the static scope of a class is the type parameter scope (\ref{generics}) of the class.
-\item An \IndexCustom{instance scope}{scope!instance}.
-The enclosing scope of a class' instance scope is the class' static scope.
+  The enclosing scope of the type-parameter scope of a class declaration is
+  the current scope of the class declaration
```

Comment:
The "current scope" confuses me here&mdash;"current" suggests variability over time. 
Maybe: "the scope containing the class declaration"?

Reply:
True, it sounds like a temporal association, but it is the standard phrase that we've always used to denote the innermost scope that contains a given syntactic entity.

In this case we don't have to use that phrase, because the current scope for a class declaration is always the library scope. So I just changed it to directly refer to the library scope, normatively.

This breaks if we introduce nested classes, of course, but we'll probably be able to fix it again if needed.